🐛 BUG: Starter pack for docs contains invalid template reference

### What package manager are you using?

npm

### What operating system are you using?

Mac

### Describe the Bug

After setting up a fresh example using the `docs` template and running it with `npm run dev`, the generated website does not work.

The error is: `Error: Cannot find module '~/layouts/MainLayout.astro'`

This is caused by the import of the template within the `md` files.
Changing `layout: ~/layouts/MainLayout.astro` to `layout: ../../layouts/MainLayout.astro` fixes the issue.

### Steps to Reproduce

1. `npm init astro` using template docs
2. `npm install`
3. `npm run dev`
4. Open website (http://localhost:3000/en/page-2)
5. `Error: Cannot find module '~/layouts/MainLayout.astro'`, expected behaviour: See the documentation website

### Link to Minimal Reproducible Example (Optional)

_No response_

--- a/examples/docs/src/pages/en/introduction.md
+++ b/examples/docs/src/pages/en/introduction.md
@@ -1,6 +1,6 @@
 ---
 title: Introduction
-layout: ~/layouts/MainLayout.astro
+layout: ../../layouts/MainLayout.astro
 ---
 
 **Welcome to Astro!**

--- a/examples/docs/src/pages/en/page-2.md
+++ b/examples/docs/src/pages/en/page-2.md
@@ -1,6 +1,6 @@
 ---
 title: Page 2
-layout: ~/layouts/MainLayout.astro
+layout: ../../layouts/MainLayout.astro
 ---
 
 Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

--- a/examples/docs/src/pages/en/page-3.md
+++ b/examples/docs/src/pages/en/page-3.md
@@ -1,6 +1,6 @@
 ---
 title: Page 3
-layout: ~/layouts/MainLayout.astro
+layout: ../../layouts/MainLayout.astro
 ---
 
 This is a fully-featured page, written in Markdown!

--- a/examples/docs/src/pages/en/page-4.md
+++ b/examples/docs/src/pages/en/page-4.md
@@ -1,6 +1,6 @@
 ---
 title: Page 4
-layout: ~/layouts/MainLayout.astro
+layout: ../../layouts/MainLayout.astro
 ---
 
 This is a fully-featured page, written in Markdown!

